app/controllers/dashboards_controller.rb

```diff
@@ -18,10 +18,10 @@ def show
 
   # Don't know how to properly test this feature, ask for dans help, should also be refactored into it's own rest model
   def send_flow
-    flow = Flowdock::Flow.new(:api_token => ENV['LP_FLOW'],
-        :source => "LaunchPairs", :from => {:name => current_user.first_name, :address => current_user.email})
+    flow = Flowdock::Flow.new(api_token: ENV['LP_FLOW'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.